checks/linux/password_manager.go

```diff
@@ -77,33 +77,26 @@ func checkForBrowserExtensions() bool {
 }
 
 func isPackageInstalled(pkgName string) bool {
+	output, err := shared.RunCommand("which", pkgName)
+	if err == nil && !strings.Contains(output, "not found") {
+		log.Debug("Package found in PATH: " + pkgName)
+		return true
+	}
+
 	pkgManagers := make(map[string]string)
 
 	// Check which package managers are available
-	if _, err := shared.RunCommand("which", "dpkg"); err == nil {
-		pkgManagers["apt"] = "dpkg -l"
-		log.Debug("apt package manager found")
-	}
+	// We only need to check for package managers that don't inject
+	// the binaries into the $PATH. Those that do are not needed
+	// because we always first check the $PATH with `which`
```

Comment:
Usually on linux these packages are not injected into path as they are not meant to run via console. They are installed under /opt/ /srv/ and only exposed via .desktop file. IF we search in package managers it is more likely to detect them than in path. For example 1password is one of such cases https://downloads.1password.com/linux/debian/amd64/stable/1password-latest.deb

Reply:
I think we should actually add more ways to detect not less, path is only one of them.

Reply:
Yes, I agree, I'll rewrite it.